include/ocpp/v201/ocpp_types.hpp

```diff
@@ -943,7 +2132,7 @@ void from_json(const json& j, Transaction& k);
 std::ostream& operator<<(std::ostream& os, const Transaction& k);
 
 struct Firmware {
-    CiString<512> location;
+    CiString<2000> location;
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1

Reply:
Accept